config/README.md

```diff
@@ -283,3 +283,78 @@ For Bazel the equivalent build command is:
 ```cli
 bazel build //... --action_env=IOX2_CUSTOM_PAL_CONFIG_PATH=/my/funky/platform --//:custom_pal_config="on"
 ```
+
+## Custom Platform Abstraction Layer POSIX
```

Comment:
@dkroenke One question, is it expected that these custom platforms work with the existing `posix.h` file in `iceoryx2-bb/posix/src/c` or should it be possible for users to provide their own header to only include the APIs that their custom platform uses?